Clients/src/presentation/components/RiskLevel/RiskChip.tsx

```
@@ -4,31 +4,32 @@ import React from "react";
 import { getSeverityColorByText, getRiskChipStyle } from "./constants";
 
 interface RiskChipProps {
-  label: string;
+  label?: string;
 }
 
 const RiskChip: React.FC<RiskChipProps> = React.memo(({ label }) => {
+  // Return chip if label is provided
+  if (label) {
+    const normalizedLabel = label?.toLowerCase().trim() ?? '';
+    const backgroundColor = useMemo(
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)